release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1065,6 +1065,8 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 [id="ocp-4-9-console-storage-bug-fixes"]
 ==== Console Storage Plug-in
 
+* Previously, when working with Ceph storage, the Console Storage Plugin unnecessarily included a redundant use of a namespace parameter. This bug had no customer-visible impact, however the plugin has been updated to avoid the redundant use of the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1982682[*BZ#1982682*])
```

Comment:
To be consistent with the subheader and ISG:
```suggestion
* Previously, when working with Ceph storage, the Console Storage Plug-in unnecessarily included a redundant use of a namespace parameter. This bug had no customer-visible impact; however, the plug-in has been updated to avoid the redundant use of the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1982682[*BZ#1982682*])
```